monkey/common/agent_configuration/default_agent_configuration.py

```diff
@@ -43,15 +43,13 @@
     8080,
     8088,
     8983,
-    9200,
     9600,
 )
 
 TCP_SCAN_CONFIGURATION = TCPScanConfiguration(timeout=3.0, ports=TCP_PORTS)
 ICMP_CONFIGURATION = ICMPScanConfiguration(timeout=1.0)
-HTTP_PORTS = (80, 443, 7001, 8008, 8080, 8983, 9200, 9600)
+HTTP_PORTS = (80, 443, 7001, 8008, 8080, 8983, 9600)
```

Comment:
Do any blackbox tests need to be updated?

Reply:
No, nothing uses it